pom.xml

```diff
@@ -62,7 +62,7 @@
 
   <properties>
     <javaVersion>8</javaVersion>
-    <mavenVersion>3.2.5</mavenVersion>
```

Comment:
notice that "require Maven" only counts for plugins
for components like this one, it's just a dependency update like any other